server/db/driver/pg/internal/matches.go

```diff
@@ -92,7 +92,7 @@ const (
 			$2, $3,
 			$4, $5,
 			$6, $7,
-			$8, $9, $10) ` // status should be MatchComplete although there is no swap
+			$8, $9, $10) `  // status should be MatchComplete although there is no swap
```

Comment:
These formatting changes aren't related to this PR, but `gofmt` picks them up, 

if you don't mind I'd fix these too so I don't have to watch out for these changes on each PR I make. Or do you recommend something else ?

Reply:
Never mind, looks like it has to do with the differences in `gofmt` behavior in go1.19 and go1.20, CI linter failed and after I've upgraded to go1.20 it doesn't try to adjust that formatting,

maybe related (for reference): https://github.com/golangci/golangci-lint-action/issues/535.